src/OpalCompiler-Core/OpalCompiler.class.st

```diff
@@ -437,21 +437,19 @@ OpalCompiler >> install [
 	method := self compile.
 	method ifNil: [ ^ method ].
 
-	protocolName ifNil: [ protocolName := Protocol unclassified ].
+	protocol ifNil: [ protocol := Protocol unclassified ].
+	protocol := class ensureProtocol: protocol.
```

Comment:
maybe `ensureProtocol: nil` could return an unclassified protocol? Or maybe not, it's a little weird. I don't know

Reply:
I just did that, thanks for the idea